### PR TITLE
external link component

### PR DESCRIPTION
**Pull request by pavloburchak**
This PR...

## Changes
- added External link component
- verify it is used in all places and behaviour it the same

## Fixes

-

## How to test it

-

## screenshots

-

## Checklist

- [ ] tested locally
- [ ] added new dependencies
- [ ] updated the docs
- [ ] added a test

### CODE REVIEW

File: src/components/molecules/EndpointModal/EndpointModal.tsx [hunk 1]
```diff
@@ -59,13 +61,9 @@ const EndpointModal: React.FC<EndpointModalProps> = props => {
           <Text>
             We could not detect the right Testkube API endpoint for you. Please enter the API endpoint for your
             installation (e.g. from the output of the Testkube installer)&nbsp;
-            <a
-              href="https://kubeshop.github.io/testkube/UI/#ui-results-endpoint"
-              target="_blank"
-              rel="noopener noreferrer"
-            >
+            <ExternalLink href="https://kubeshop.github.io/testkube/UI/#ui-results-endpoint">
```
Comment (rangoo94) [inline src/components/molecules/EndpointModal/EndpointModal.tsx:64]: Missing `rel="noopener noreferrer"`

**Review by rangoo94 — APPROVED** [review]
I don't think that such component is needed (for me `<a target='_blank'>` is fine and simpler to customize), but if you prefer to use it instead of `a[target=_blank]` it's fine for me :)

File: src/components/molecules/EndpointModal/EndpointModal.tsx (hunk 1 above)
Comment (pavloburchak) [reply src/components/molecules/EndpointModal/EndpointModal.tsx:64]: it is not needed here

**Comment by pavloburchak** [comment]
> I don't think that such component is needed (for me `<a target='_blank'>` is fine and simpler to customize), but if you prefer to use it instead of `a[target=_blank]` it's fine for me :)

I would like to extend it in future. And for me it is just more intuitive to customise it

**Pull request merged**